flags/settings.py

```diff
@@ -0,0 +1,43 @@
+import logging
+
+from importlib import import_module
+
+from django.conf import settings
+
+logger = logging.getLogger(__name__)
+
+# Private set of all global flags from sources in FLAG_SOURCES. This is
+# populated on AppConfig.ready() by add_flags_from_sources below.
+SOURCED_FLAGS = {}
+
+
+def add_flags_from_sources(sources=None):
+    """ Read flags from sources defined in settings.FLAG_SOURCES.
+    FLAG_SOURCES is expected to be a list of Python module with flags
+    specified with variable assignment, e.g. MY_FLAG = True """
+    global SOURCED_FLAGS
+
+    if sources is None:
+        sources = getattr(settings, 'FLAG_SOURCES', ())
+
+    for source_str in sources:
+        source = import_module(source_str)
+        for flag in (f for f in dir(source)
+                     if f.isupper() and isinstance(getattr(source, f), bool)):
+            SOURCED_FLAGS[flag] = getattr(source, flag)
```

Comment:
Should this raise an exception if a flag name is duplicated across sources? This seems like something that could be sneaky and hard to debug if it happened.